3.-Web-api-call-Microsoft-graph-for-personal-accounts/TodoListClient/MainWindow.xaml.cs

```diff
@@ -362,6 +362,7 @@ private async void SignIn(object sender = null, RoutedEventArgs args = null)
                 var result = await _app.AcquireTokenInteractive(Scopes)
                     .WithAccount(accounts.FirstOrDefault())
                     .WithPrompt(Prompt.SelectAccount)
+                    .WithExtraScopesToConsent(new[] { "user.read" })
```

Comment:
The comment is very out of date - it's ADAL specific. It would also help to add more details in the comment on why we need extra scopes and why they can't go into Scopes

Reply:
"the comment"? which comment @bgavrilMS ?

Reply:
The comment above this method call. Line 360